pl_examples/hydra_examples/cpu_template.py

```diff
@@ -0,0 +1,40 @@
+"""
+Runs a model on the CPU on a single node.
+"""
+
+import hydra
+
+import pytorch_lightning as pl
+from omegaconf import DictConfig
+from pl_examples.hydra_examples.pl_hydra.trainer_conf import init_trainer
+import pl_examples.hydra_examples.conf.optimizer
+import pl_examples.hydra_examples.conf.scheduler
```

Comment:
run flake8, I think you have some unused imports.

Reply:
These are actually used to register the structured configs used by Hydra

Reply:
oh, I see.
that's one of the things I am not quiet sure about how to do.
What might make sense is to put all the cs.store() calls in `conf/__init__.py` and include just that, but there are no established best practices for this yet.

Reply:
Done